agents/cryomech-cpa1114/cryomech_cpa1114_agent.py

```diff
@@ -35,7 +35,73 @@ def signal_handler(signum, frame):
         yield
     finally:
         signal.alarm(0)
+
+class PTC:
+    def __init__(self, ip_address, port=502, timeout=10):
+        self.ip_address = ip_address
+        self.port = port  
+        
+        self.comm = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.comm.connect((self.ip_address, self.port))
+        self.comm.settimeout(timeout)
+                 
+    def get_data(self):
+        """
+        Gets the raw data from the ptc and returns it in a usable format. 
+        """
+        self.comm.sendall(self.buildRegistersQuery()) 
+        data = self.comm.recv(1024)
+        brd = self.breakdownReplyData(data)
+            
+        return brd    
+    
+    def buildRegistersQuery(self):
+        query = bytes([0x09, 0x99,  # Message ID
```

Comment:
I think using [struct.pack](https://docs.python.org/3.7/library/struct.html#struct.pack) here is clearer and more maintainable than setting each individual byte manually.

Reply:
Didn't make this change after mutual agreement